app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -14,12 +14,7 @@ import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.Button
-import android.widget.LinearLayout
-import android.widget.TextView
-import android.widget.Switch
-import android.widget.Toast
-import androidx.activity.OnBackPressedCallback
+import android.widget.*
```

Comment:
for optimizing compiler speed, I will be replacing these import all dependencies to be more specific

Reply:
Is there any specific time we should use the * (wildcard)?

Because if I were to remove it then i'd have to add the following import:

import android.widget.Button
import android.widget.LinearLayout
import android.widget.TextView
import android.widget.Toast
import android.widget.Switch

Reply:
Yes, when you notice that the library is small and you use them sparingly, then it won't be so bad. In this case, there is a lot of stuff in the widget package, so when you use the wild card, every time there is an unknown variable, it will check it against every element in the package of every package with a wildcard until it finds a match. Eliminating the number of checks will be important for it to be sped up

Reply:
Alrighty, in that case il follow your suggestion! Thank you 👯‍♂ 👍